ui/app/mainui/AppMain.qml

```diff
@@ -1339,10 +1341,19 @@ Item {
         }
     }
 
+    Connections {
+        target: appMain.rootStore.mainModuleInst
+        function onActiveSectionChanged() {
+            rootDropAreaPanel.activeChatType = appMain.rootStore.mainModuleInst.getCommunitySectionModule().activeItem.type
+        }
+    }
+
     DropAreaPanel {
+        id: rootDropAreaPanel
         width: appMain.width
         height: appMain.height
-        activeChatType: chatCommunitySectionModule && chatCommunitySectionModule.activeItem.type
+        activeChatType: appMain.rootStore.mainModuleInst.getCommunitySectionModule() ? appMain.rootStore.mainModuleInst.getCommunitySectionModule().activeItem.type
```

Comment:
Probably there is a way how to access activeItem from communitySection in declarative way and get rid of connections